src/rules/noUnboundMethodRule.ts

```diff
@@ -20,22 +20,55 @@ import * as ts from "typescript";
 import * as Lint from "../index";
 
 const OPTION_IGNORE_STATIC = "ignore-static";
+const OPTION_WHITELIST = "whitelist";
+
+const OPTION_WHITELIST_EXAMPLE = [
+    true,
+    {
+        [OPTION_WHITELIST]: ["expect", "typeof"],
+    },
+];
 
 interface Options {
     ignoreStatic: boolean;
+    whitelist: string[];
```

Comment:
Looks like this is only ever checked for a name to exist, so it'd be a little more efficient to use a `Set<string>` and `ctx.options.whitelist.has(callingMethodName)`.

Reply:
Changed to `Set`